packages/salesforcedx-utils-vscode/src/index.ts

```diff
@@ -1,9 +1,29 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
 export {
-  SFDX_PROJECT_FILE,
-  ENV_SFDX_DEFAULTUSERNAME,
-  ENV_SFDX_INSTANCE_URL,
-  SFDX_CONFIG_ISV_DEBUGGER_SID,
-  SFDX_CONFIG_ISV_DEBUGGER_URL,
-  DEFAULT_CONNECTION_TIMEOUT_MS,
-  CLIENT_ID
-} from './constants';
```

Comment:
Moving these exports to `src/types`. Tests of `salesforcedx-sobjects-faux-generator` depends on it, and it doesn't have vscode dependency. Move them so we don't have to create similar mocks in `salesforcedx-sobjects-faux-generator`.